_drafts/SC2-4-2-title.md

```diff
@@ -1,126 +1,52 @@
 ---
-rule_id: SC2-4-2-title
-name: Page Title
-test_mode: semi-automatic
+name: HTML page has a title
+description: |
+  This rule checks that the HTML page has a title
 
-criteria:
-- 2.4.2 # Page Titled (Level A)
+success_criterion:
+- 2.4.2 # Page Titled
 
-authors:
+test_aspects:
+- DOM Tree
 
+authors:
+- Wilco Fiers
+- Stein Erik Skotkjerra
+- Bryn Anderson
+- Anne Thyme Nørregaard
+- Jey Nandakumar
 ---
 
-## Description
-
-This test checks that the page title (`<title` element in the head section of a webpage) a. exists, and b. identifies the role or content of the web page.
-
-## Background
-
-- [G88: Providing descriptive titles for Web pages](http://www.w3.org/TR/WCAG20-TECHS/G88.html)
-- [H25: Providing a title using the title element](http://www.w3.org/TR/WCAG20-TECHS/H25.html)
-- [F25: Failure of Success Criterion 2.4.2 due to the title of a Web page not identifying the contents](http://www.w3.org/TR/WCAG20-TECHS/F25.html)
-## Assumptions
-- Technique ARIA 1 is not considered for this test, although it was added in a recent [update of the Understanding WCAG 2.0 document](http://www.w3.org/TR/2014/NOTE-UNDERSTANDING-WCAG20-20140408/complete-diff.html).
-- If there are more than one `<title>` elements this test will throw an error.
-
-## Test properties
-
-| Property          | Value
-|-------------------|----
-| Test name         | Page Title
-| Test requirement  | 2.4.2 Page Titled
-| Test mode         | semiAuto
-| Test environment  | HTML source or DOM
-| Test subject      | single web page
-
 ## Test procedure
 
-### Selector
-
-Test mode: [automatic][AUTO]
-
-Select the `head` element
-
-### Step 1
-
-Test mode: [automatic][AUTO]
-
-Check that there is a `title` element in the html document.
+### Applicability
 
-If no, return
+The rule applies to any page where the root element is an `html` element, and which is not embedded in another page.
 
-| Outcome  | Failed
-|----------|-----
-| Testcase | SC2-4-2-title
-| ID       | SC2-4-2-title-fail1
-| Error    | Missing title element
+**Note**: Pages may be embedded inside other pages through elements such as iframes and object elements.
 
-Else, continue with [Step 2](#step-2)
+### Expectation 1
 
-### Step 2
+The page contains at least one `title` element.
 
-Test mode: [automatic][AUTO]
+**Note**: The `title` element exists in other namespaces such as SVG. These are not `title` elements for HTML document and should be ignored.
```

Comment:
Might be worth also calling out the scenario where the content of an iframe has a title element, as also something to be ignored, like the SVG title elements.